### PR TITLE
Update Specs

### PR DESCRIPTION
**Pull request by michaelwehrley**
* Add spec helpers for feature specs
* Add "Definition of Increased Risk" Spec

### CODE REVIEW

File: spec/support/feature_helpers.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module FeatureHelpers
```
Comment (ericcf) [inline spec/support/feature_helpers.rb]: I suggest using a page object model pattern here rather than a global method.

Comment (michaelwehrley) [inline spec/support/feature_helpers.rb]: I don't quite follow, do you have an example I can look at?  Thanks.

Comment (ericcf) [inline spec/support/feature_helpers.rb]: I guess this isn't a global method, but it still doesn't feel like a use case for inheritance to me.

Why not something like: 
```ruby
class LoginPage
  attr_reader :page

  def initialize(page)
    @page = page
  end

  def visit
    page.visit "/index.html"

    self
  end

  def log_in_and_begin
    page.fill_in "Patient ID", with: "test"
    page.click_on "Begin"
  end
end
```

which you can use in your specs

```ruby
login_page = LoginPage.new(page).visit
login_page.log_in_and_begin
```

It makes it more object oriented (composition over inheritance), and more explicit.

Comment (michaelwehrley) [inline spec/support/feature_helpers.rb]: perfect!  Thanks, now I understand.

**Comment by michaelwehrley** [comment]
@ericcf let me know what you think.  Thanks!

**Comment by ericcf** [comment]
This looks like you came up with a different pattern than the page object model. How did you decide on it?

**Comment by ericcf** [comment]
Nice! 🚀

**Pull request merged**